site/en/docs/extensions/mv3/declare_permissions/index.md

```diff
@@ -8,33 +8,36 @@ description: An overview of the valid values for the permissions property in man
 ---
 
 To use most [Chrome APIs][api-ref], your extension must declare its intent in the permissions fields
-of the [manifest](#manifest). Extensions can request four categories of permissions, specified using the respective keys in the manifest:
+of the [manifest](#manifest). Extensions can request the following categories of permissions,specified using the respective manifest keys:
 
-`"permissions"`
-: contain items from a list of known strings (such as `"geolocation"`).
+[`"permissions"`](#permissions)
+: contain items from a list of known strings (such as `"geolocation"`). Changes may trigger a [warning](#warnings).
 
-`"optional_permissions"`
-: are like regular `permissions`, but are granted by the extension's user at runtime, rather than in advance.
+[`"optional_permissions"`][api-perms]
+: are like regular `permissions`, but are granted by the user at runtime, rather than in advance.
 
-`"host_permissions"`
-: contain one or more [match patterns][doc-match] that give access to one or more hosts.
+[`"content_scripts.matches"`][doc-cs-static]
+: contain one or more [match patterns][doc-match] that allows content scripts to inject into one or more hosts. Changes may trigger a [warning](#warnings).
+
+[`"host_permissions"`](#host-permissions)
+: contain one or more [match patterns][doc-match] that give access to one or more hosts. Changes may trigger a [warning](#warnings).
 
 `"optional_host_permissions"`
-: are like regular `host_permissions`, but are granted by the extension's user at runtime, rather than in advance.
+: are like regular `host_permissions`, but are granted by the user at runtime, rather than in advance.
```

Comment:
```suggestion
: are like regular `host_permissions`, but are granted by the user at runtime, rather than at install time.
```

Not sure about this